### PR TITLE
Replace indiesdev/curl@v1.1 action

### PR DESCRIPTION
**Pull request by ponny-m**
## What was the problem?[^1]
Multiple warnings with `indiesdev/curl@v1.1` action in our pipelines:
- `The following actions uses node12 which is deprecated and will be forced to run on node16: indiesdev/curl@v1.1.`
- `The set-output command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/`

Github Actions workflows affected by warnings above:
1. **Continuous Integration (CI)** => 26 warnings across various jobs
2. **CI: API Gateway integration test** => 10 warnings (mostly deprecated 'set-output' command)
3. **CI: BIE Kafka End-2-End Test** => 2 warnings
4. **CI: EE - EP Merge - Integration Test** => 4 warnings
5. **CI: SVC BGS API integration test** => 9 warnings
6. **CI: SVC BIP API integration test** => 2 warnings
7. **Secrel** => 6 warnings

Associated tickets or Slack threads:
- #2721 

## How does this fix it?[^1]
Created a new action found in a [public VA repo](https://github.com/department-of-veterans-affairs/curl-action)

## How to test this PR
- Run the affected workflows with my branch (update-action/ponnia) and errors related to this action don't pop up anymore.
- [Secrel](https://github.com/department-of-veterans-affairs/abd-vro-internal/actions/runs/8549531195) => For clarity, all indiesdev/curl@v1.1 action warnings in our Secrel workflow have been resolved. However, the current node.js 16 warnings seen here are from another action (see picture below). Based on this [conversation](https://dsva.slack.com/archives/C04QLHM9LR0/p1710529783156679) reverting back to v5 brought back these errors. 
Screenshot 2024-04-04 at 1 24 09 AM

### CODE REVIEW

File: helm/svc-bie-kafka/values.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ ports:
 
 livenessProbe:
   httpGet:
-    path: /actuator/health/liveness
+    path: /actuator/health/liveness
```
Comment (tejans24) [inline helm/svc-bie-kafka/values.yaml]: is the empty space and intended change here?

File: .github/workflows/api-gateway-integration-test.yml [hunk 2]
```diff
@@ -65,57 +65,57 @@ jobs:
 
       - name: "Swagger UI - Request"
         id: "swaggerUI"
-        uses: indiesdev/curl@v1.1
+        uses: Ponnia-M/curl@v1.0.0
```
Comment (tejans24) [inline .github/workflows/api-gateway-integration-test.yml]: @Ponnia-M should this be within Department of VA GitHub repo rather than your personal repo?

**Review by tejans24 — COMMENTED** [review]
Adding comments

File: .github/workflows/api-gateway-integration-test.yml (hunk 2 above)
Comment (ponny-m) [reply .github/workflows/api-gateway-integration-test.yml]: @tejans24 I don't have access to fork it into the Department of VA GitHub repo

File: helm/svc-bie-kafka/values.yaml (hunk 1 above)
Comment (ponny-m) [reply helm/svc-bie-kafka/values.yaml]: No that wasn't intended

**Comment by ponny-m** [comment]
@tejans24 @Cheng PR has been updated and curl-action is now in [public VA repo](https://github.com/department-of-veterans-affairs/curl-action/tree/v1.0.0). Testing was successful for affected workflows.

**Review by chengjie8 — APPROVED** [review]
LGTM

**Pull request merged**